src/lib.rs

```diff
@@ -326,6 +326,8 @@ if_hyper! {
     #[cfg(feature = "multipart")]
     pub use self::async_impl::multipart;
 
+    #[cfg(feature = "__tls")]
+    pub use self::tls::TlsInfo;
```

Comment:
I think I'd suggest leaving this re-export out, and then I think we can ship this!